src/hdf5_time_series_storage.jl

```diff
@@ -115,6 +115,27 @@ function serialize_time_series!(
     return
 end
 
+function get_data_type(ts::TimeSeriesData)
```

Comment:
we should add a comment here that this function is implemented this way to avoid running eval on arbitrary code from the HDF5

Reply:
ok, I'll add a comment there